plugin_manager.py

```diff
@@ -268,6 +269,33 @@ async def update_plugins(self):
                 plugins_to_update.append(plugin.update())
         await asyncio.gather(*plugins_to_update)
 
+    async def get_new_plugins(self, new_plugin_count):
+        valid_new_plugins = []
+        plugin_dictionary, sorted_plugins, new_plugins = {}, {}, {}
+        count = 0
+        await self.plugin_manager.setup_index()
+        all_plugins = await self.plugin_manager.categories["All"].get_plugins()
+        for plugin in all_plugins:
+            plugin_dictionary[plugin.name] = plugin.info
+        for key, value in plugin_dictionary.items():
+            latest_version = max(value['versions'].keys())
+            new_plugins[key] = {'released_on': value['versions'][latest_version]['released_on'],
+                                'api_version': value['versions'][latest_version]['api_version']}
+            sorted_plugins = dict(sorted(new_plugins.items(),
+                                         key=lambda x: datetime.strptime(
+                                             x[1]['released_on'], '%d-%m-%Y'),
+                                         reverse=True))
+
+        for key, value in sorted_plugins.items():
+            if count < new_plugin_count:
```

Comment:
Need help in here to check the api_version. It is present in the dictionary values, but I am not sure how check that with the device.